### ISSUE TITLE
Comment Projections

### ISSUE THREAD

**Issue by mbjorkegren** [pol-is/issues#50]
Compute the projection of comments using the center of mass, and apply a scaling factor to the projection.

One way to structure that would be to do it just like like `math_main > pca > comps`, but call it `comment-projection`:
`math_main > pca > comment-projection > 0`
`math_main > pca > comment-projection > 1`

One idea for selecting the scaling factor is to scale all comments up so that they extend as far as possible without extending beyond the most extreme participants.

**Comment by patcon** [comment]
Hi! Is this closed because done, or something else?

**Comment by metasoarous** [comment]
Yes, this has been implemented in the math. If you look at the output of `https://pol.is/api/v3/math/pca2?conversation_id=<blah>`, the comments projection can be found under the `pca` key. We decided against wiring this up to the participation viz however.

**Comment by colinmegill** [comment]
If I've got the right API (I think I do...) this is responsible for the
numbers in the plot in the report
